src/screens/main/profile/ThemeSwitch.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext, useState } from 'react'
```

Comment:
Have in mind that we intentionally separated 
`src/screens` and `src/components`

This is obviously a presentational component so should be moved to `src/components/shared`